fastlane/Fastfile

```diff
@@ -24,11 +24,15 @@ files_with_version_number_without_prerelease_modifiers = {
     './ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Info.plist' => PLIST_VERSION_PATTERNS,
 }
 POD_VERSION_PATTERNS = ['\'RevenueCat\', \'{x}\'', '\'RevenueCatUI\', \'{x}\'']
+SPM_VERSION_PATTERNS = ['.package(url: "https://github.com/RevenueCat/purchases-ios-spm", exact: "{x}"),']
+SPM_RESOLVED_VERSION_PATTERNS = ['        "version" : "{x}"']
```

Comment:
do we need all the whitespace before the version here?

Reply:
Not necessarily. The intention was to make the pattern as strict as possible, but it works without it too. Removed here: https://github.com/RevenueCat/purchases-hybrid-common/pull/922/commits/13b2b231ada16b2852f6e7bc95b479d2acbac15f.